src/components/map-container.tsx

```diff
@@ -45,33 +44,30 @@ import {
   LayerBaseConfig,
   VisualChannelDomain
 } from '@kepler.gl/layers';
-import {setLayerBlending} from 'utils/gl-utils';
-import {transformRequest} from 'utils/map-style-utils/mapbox-utils';
+import {Filter, InteractionConfig, MapState, SplitMapLayers} from '@kepler.gl/types';
 import {
-  getLayerHoverProp,
-  renderDeckGlLayer,
-  prepareLayersToRender,
-  prepareLayersForDeck,
-  LayerHoverProp
-} from 'utils/layer-utils';
+  errorNotification,
+  setLayerBlending,
+  transformRequest,
+  observeDimensions,
+  unobserveDimensions
+} from '@kepler.gl/utils';
 
 // default-settings
 import {ThreeDBuildingLayer} from '@kepler.gl/deckgl-layers';
 import {FILTER_TYPES, GEOCODER_LAYER_ID, THROTTLE_NOTIFICATION_TIME} from '@kepler.gl/constants';
 
 import ErrorBoundary from 'components/common/error-boundary';
-import {observeDimensions, unobserveDimensions} from '../utils/observe-dimensions';
 import {LOCALE_CODES} from '@kepler.gl/localization';
+import {MapControls, MapStyle, Viewport} from 'reducers';
+import {Datasets} from 'reducers/table-utils';
 import {
-  Datasets,
-  Filter,
-  InteractionConfig,
-  MapControls,
-  MapState,
-  MapStyle,
-  Viewport
-} from 'reducers';
-import {SplitMapLayers} from 'reducers/vis-state-updaters';
+  getLayerHoverProp,
+  LayerHoverProp,
+  prepareLayersForDeck,
+  prepareLayersToRender,
+  renderDeckGlLayer
+} from 'reducers/layer-utils/layer-utils';
```

Comment:
will you edit the path when isolation reducers? i feel like this should be either
'reducers/layer-utils/layer-utils` or `reducers/utils/layer-utils`